db.json

```
@@ -147,6 +147,20 @@
         }
       ],
       "standards": [
+        {
+          "name": "Cookie Store API",
+          "href": "https://wicg.github.io/cookie-store/",
+          "description": "An asynchronous Javascript cookies API for documents and service workers.",
+          "authors": [
+            {
+              "name": "Dylan Cutler",
+              "email": "dylancutler@google.com"
+            }
+          ],
+          "reference": "COOKIESTOREAPI",
```

```suggestion
          "reference": "COOKIESTORE",
```

is more congruent with other specifications.